module_utils/fdm_swagger_client.py

```diff
@@ -43,7 +43,7 @@ class SpecProp:
 
 class PropName:
     ENUM = 'enum'
-    TYPE = 'type'
+    TYPE = 'type'  # Not present since FTD 6.6.0 release
```

Comment:
This comment is not very accurate as it's absent only for object-type fields, for other fields (boolean, string, etc) it's still present, so I'd remove it.

Reply:
I've reverted this change. @annikulin PTAL.